week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -17,19 +17,24 @@ Passwords must not be any previous password in the passwords array.
 Example 1:
 PreviousPassword = ["fhD8!yrjj", "ttkTu.wer3", "dvyyeyY!5", "qwbfj76%", "tytT3729."];
 
+// dvyyeyY!5  is passing the test - length, uppercase, lowercase, numbers, symbols, why is it false?
+
+
 Expected Result:
 PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
 function validatePasswords(passwords) {
-
+  
+return   passwords.map(password => password.length >=5 && /[A-Z]/g.test(password)  && /[a-z]/g.test(password)  && /[0-9]/g.test(password)  && /[^a-zA-Z\d\s:]/g.test(password) )
```

Comment:
I think this is pretty hard to read quickly. How could you format this code so it's a bit easier to quickly understand it?